docs/chapters/05-2.md

```diff
@@ -0,0 +1,211 @@
+---
+lang-ref: ch.05-2
+title: Optimization Techniques II
+authors: Guido Petri, Haoyue Ping, Chinmay Singhal, Divya Juneja
+date: 29 February 2020
+typora-root-url: 05-3
+---
+
```

Comment:
Add an extra empty line before every title, so that there will be in total 2 empty lines.